.github/ISSUE_TEMPLATE.md

```diff
@@ -23,8 +23,8 @@ Create a [gist](https://gist.github.com) which is a paste of your _full_ Appium
 
 ## Code To Reproduce Issue [ Good To Have ]
 
-Please remember that, with sample code; it's easier to reproduce bug and much faster to fix it.
+Please remember that, with sample code; it's easier to reproduce the bug and it's much faster to fix it.
```

Comment:
Since we're here, can you get rid of the comma before and the semicolon after "with sample code"?